### PR TITLE
Add operating system & web browser fields for bug report form

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
This PR add 2 fields for bug report form. One of them is for operating system and the other for web browser.

image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -2,6 +2,22 @@ name: Bug report
...
 labels: ["bug"]
 body:
+  - type: dropdown
+    attributes:
+      label: Operating System
+      description: On which operating system do you have a problem?
+      options:
+        - Windows
+        - MacOS
+        - Linux
```
Comment (OroArmor) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:12]: I would also include mobile os (android and ios) and an option for other that they can specify.

**Review by OroArmor — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (Mysterious-Dev) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:12]: I got this with your request. I don't know if ot's possible to hide/show other field dynamically :/

image

**Comment by Mysterious-Dev** [comment]
Any news ?

**Comment by gdude2002** [comment]
I'm not familiar with how these forms work - is it possible to only show "Other" when it's selected in the dropdown at all?

**Comment by OroArmor** [comment]
Let me check the documentation

**Comment by OroArmor** [comment]
It seems github just uses a normal text area. https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

**Comment by Mysterious-Dev** [comment]
> I'm not familiar with how these forms work - is it possible to only show "Other" when it's selected in the dropdown at all?

I don't think that's possible :/

**Comment by gdude2002** [comment]
In that case, let's just stick with the free-form input box, like Oro suggested.

**Comment by Mysterious-Dev** [comment]
It's done, I modify this detail ^^

**Pull request merged**